City-of-Helsinki/kukkuu-admin
# KK-594 | Add edit message view

## Description

Implements the update message and get message APIs. Provides the UI layer for editing a message.

I also removed a default value from the list view that I had forgot to remove earlier.

## Context

After this change users are able to edit messages in the admin UI.

[KK-594](https://helsinkisolutionoffice.atlassian.net/browse/KK-594)

## How Has This Been Tested?

This has been tested by hand.

## Manual Testing Instructions for Reviewers

As a logged in admin user

1. Access messages from the left navigation menu
1. Select a message from the list by clicking it
1. Remove `/show` from the url
1. Expect to arrive in a view like in the screenshot
1. Expect to be able to change fields
1. Expect changes to persist after a successful save
1. Expect delete button to *not* work

## Screenshots

<img width="1680" alt="Screenshot 2020-10-21 at 15 27 46" src="https://user-images.githubusercontent.com/9090689/96719402-0261c380-13b2-11eb-9927-4a727418534b.png">
